nmsg/input.c

```diff
@@ -519,3 +592,17 @@ input_flush(nmsg_input_t input) {
 
 	return (nmsg_res_success);
 }
+
+void
+_input_stop(nmsg_input_t input) {
```

Comment:
Kafka consumer is in infinite loop when it reads newest messages. It needs to be signaled to stop before nmsg reaches nmsg_io_destroy. Otherwise the only way to stop nmsgtool that consumes Kafka messages is kill -9

Reply:
The kafka consumer _shouldn't_ be in an infinite loop. Like socket and zeromq inputs, it should return `nmsg_res_again` when the read times out (with `NMSG_RBUF_TIMEOUT` as the timeout).